src/platforms/python/guides/rq/index.mdx

```diff
@@ -31,6 +31,24 @@ The integration will automatically report errors from all RQ jobs.
 
 Generally, make sure that the **call to `init` is loaded on worker startup**, and not only in the module where your jobs are defined. Otherwise, the initialization happens too late and events might end up not being reported.
 
+In addition, make sure that **`init` is called once** in your app and imported from there in the different places it is needed. In a Flask app, for example, Sentry and `RqIntegration` can be initalised as 
+
+```python {filename:app.py}
+import sentry_sdk
+from sentry_sdk.integrations.flask import FlaskIntegration
+from sentry_sdk.integrations.rq import RqIntegration
+
+sentry = sentry_sdk.init(
+    dsn=PUBLIC_DSN,
+    integrations=[FlaskIntegration(), RqIntegration()])
+```
+
+`mysettings.py` then becomes
+
+```python {filename:mysettings.py}
+from app import sentry
```

Comment:
@untitaker could you please have a look? I'm afraid this doesn't match what you had in mind in the referred comments.

I'd expect an import just for the side effect of initializing the SDK, as one shouldn't need to use the return value of `sentry_sdk.init`.

Reply:
that's correct. return value of `init()` is not useful, and so this import can read `import app`

`from app import sentry` communicates intent better but that's nothing you can't fix with a code comment

Reply:
Alright, I `import app` now and have added a comment.